dashboard/client/src/pages/node/ClusterLayout.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { RiInformationLine, RiLineChartLine } from "react-icons/ri";
+import { SideTabLayout, SideTabRouteLink } from "../layout/SideTabLayout";
+
+export const ClusterLayout = () => {
+  return (
+    <SideTabLayout>
+      <SideTabRouteLink tabId="info" title="Info" Icon={RiInformationLine} />
+      <SideTabRouteLink
+        to=""
+        tabId="charts"
+        title="Charts"
+        Icon={RiLineChartLine}
```

Comment:
The icon is supposed to be a table icon right?

Also, let's change the title to "Table" and tabId to "table"